pkg/migrations/sql/idempotent/base.sql

```diff
@@ -1769,13 +1789,13 @@ BEGIN
                     FROM SCHEMA_CATALOG.label_key_position lkp
                     WHERE lkp.metric_name = m.metric_name
                     ORDER BY key) label_keys,
-                hi.total_size as size,
+                chs.uncompressed_total_bytes as uncompressed_size,
+                chs.compressed_total_bytes as compressed_size,
+                pg_size_pretty(pg_size_bytes(chs.uncompressed_total_bytes)::numeric + pg_size_bytes(chs.compressed_total_bytes)::numeric) as total_size,
```

Comment:
This is done so that the `size = compressed_size + uncompressed_size`, otherwise, fetching total size from timescsaledb_information.hypertable leads to mismatch of the sums (moreover that case had `uncompressed_size > size` which is doubtful).